docs/topics/proc-configure-launcher-sso-realm.adoc

```diff
@@ -0,0 +1,18 @@
+[[launcher-sso-settings]]
+= Configuring {launcher} {RHSSO} Federated Identity Settings
+In order to use {launcher} in {OpenShiftOnline}, you must grant it access to your GitHub and OpenShift accounts. This is a one-time action.
+
+[sidebar]
```

Comment:
This is an interesting usage, but I am afraid it deviates from the template. Do you insist on keeping it here? It probably won't harm anything now, but it may cause issues when on the CP.

Reply:
@tradej i like having the prerequisites stand out. I feel like it makes it clearer (visually) that these items *must* be done first. Would you be open to adopting this for our template?

Reply:
@rhoads-zach, I do agree it makes sense, although I'm a little conflicted because I suspect that's not the intended usage of the typographic element of _sidebar._ Anyway, for maximum consistency, let me propose this:

1) You remove the `[sidebar]` usage from this immediate PR.
2) Either myself or you immediately file a PR against `modular-docs` with the proposal and try to have it accepted. Mind you that the person from the modular docs team who has the final word is on PTO this week, so it may take a while to get it accepted.
3) Once (2) is accepted, we file a follow-up issue to reintroduce `[sidebar]` to all our (template-based) modules, not only the GSG.

What do you think?